pam/integration-tests/vhs-helpers_test.go

```diff
@@ -200,13 +200,11 @@ func newTapeData(tapeName string, outputDir string, settings ...tapeSetting) *ta
 		m[s.Key] = s.Value
 	}
 	return &tapeData{
-		Name:      tapeName,
-		OutputDir: outputDir,
-		Outputs: []string{
-			tapeName + ".txt",
-		},
-		Settings: m,
-		Env:      make(map[string]string),
+		Name:           tapeName,
+		OutputDir:      outputDir,
+		OutputFilename: tapeName + ".tape.txt",
```

Comment:
This is not a tape, right? But rather the recording.

So why `tape.txt`

Reply:
If you want to keep that form, it's more `tape. out.txt` or `tapeoutput` or something making sure this is not the tape

Reply:
I'm now just using `.txt` as the suffix